README.md

```diff
@@ -38,9 +44,15 @@ from transformers import AutoTokenizer
 import soundfile as sf
 import torch
 
-device = "cuda:0" if torch.cuda.is_available() else "cpu"
+device = "cpu"
+if torch.cuda.is_available():
+    device = "cuda:0"
+if torch.backends.mps.is_available():
+    device = "mps"
+if torch.backends.xpu.is_available():
+    device = "xpu"
 
-model = ParlerTTSForConditionalGeneration.from_pretrained("parler-tts/parler_tts_mini_v0.1").to(device)
+model = ParlerTTSForConditionalGeneration.from_pretrained("parler-tts/parler_tts_mini_v0.1").to(device, dtype=torch.bfloat16)
```

Comment:
The issue with this example is that not all hardware supports bfloat16. E.g. older V100 GPUs only support float16 and float32. If a user with this hardware tries to run the model in bfloat16, they'll hit an error.

A conservative approach to this would be to set the dtype to float16 if we're using cuda/mps, and float32 otherwise. We can do this as follows:
```python
device = "cuda:0" if torch.cuda.is_available() else "mps" if torch.backends.mps.is_available() else  "cpu"
torch_dtype = torch.float16 if device != "cpu" else torch.float32

model = ParlerTTSForConditionalGeneration.from_pretrained("parler-tts/parler_tts_mini_v0.1").to(device, dtype= torch_dtype)
```